functests/4_latency/latency.go

```diff
@@ -135,38 +140,30 @@ var _ = Describe("[performance] Latency Test", func() {
 			oslatArgs := []string{
 				fmt.Sprintf("-runtime=%s", latencyTestRuntime),
 			}
-			latencyTestPod = getLatencyTestPod(profile, workerRTNode, testName, oslatArgs)
-			createLatencyTestPod(latencyTestPod)
+			latencyTestPod = getLatencyTestPod(profile, workerRTNode, testName, oslatArgs, logName)
+			createLatencyTestPod(latencyTestPod, workerRTNode, logName)
+			logFileContent = getLogFile(workerRTNode, logName)
```

Comment:
you can just `logFileContent := getLogFile(workerRTNode, logName)`

Reply:
The same true for all other places

Reply:
okay i will change it , I thought to make use of it the same as logName.